scripts/all-command-help.js

```diff
@@ -4,99 +4,99 @@
 // Reads from dist directory, so be sure to build before running this target
 //
 
-'use strict'
-
-const _ = require('lodash')
-const fs = require('fs')
-const os = require('os')
-const path = require('path')
-const util = require('util')
-const Table = require('cli-table3')
-const { getClassHelpText, getOptionsDescription, getPositionalOptionsDescription } = require('../dist/util/commandline/option-decorators')
-const { getOptionsForTwoColumnTableWithNoBorders } = require('../dist/util/interaction/out')
+"use strict";
+
+const _ = require("lodash");
+const fs = require("fs");
+const os = require("os");
+const path = require("path");
+const util = require("util");
+const Table = require("cli-table3");
+const {
+  getClassHelpText,
+  getOptionsDescription,
+  getPositionalOptionsDescription,
+} = require("../dist/util/commandline/option-decorators");
+const { getOptionsForTwoColumnTableWithNoBorders } = require("../dist/util/interaction/out");
 function checkStats(dir, fileName, checker) {
   try {
-    let s = fs.statSync(path.join(dir, fileName))
-    return checker(s)
-  }
-  catch (err) {
-    return false
+    let s = fs.statSync(path.join(dir, fileName));
```

Comment:
looks like this should be const -- why no linting error?

Reply:
We never linted JavaScript before. I enabled it and applied fixes.